kotlin/feature-layer-query/src/main/java/com/esri/arcgisruntime/sample/featurelayerquery/MainActivity.kt

```diff
@@ -0,0 +1,174 @@
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package com.esri.arcgisruntime.sample.featurelayerquery
+
+import com.esri.arcgisruntime.data.ServiceFeatureTable
+import android.app.SearchManager
+import android.content.Context
+import android.content.Intent
+import android.graphics.Color
+import android.os.Bundle
+import android.util.Log
+import android.view.Menu
+import android.widget.Toast
+import androidx.appcompat.app.AppCompatActivity
+import androidx.appcompat.widget.SearchView
+import com.esri.arcgisruntime.concurrent.ListenableFuture
+import com.esri.arcgisruntime.data.FeatureQueryResult
+import com.esri.arcgisruntime.data.QueryParameters
+import com.esri.arcgisruntime.geometry.Point
+import com.esri.arcgisruntime.geometry.SpatialReferences
+import com.esri.arcgisruntime.layers.FeatureLayer
+import com.esri.arcgisruntime.mapping.ArcGISMap
+import com.esri.arcgisruntime.mapping.Basemap
+import com.esri.arcgisruntime.mapping.Viewpoint
+import com.esri.arcgisruntime.symbology.SimpleFillSymbol
+import com.esri.arcgisruntime.symbology.SimpleLineSymbol
+import com.esri.arcgisruntime.symbology.SimpleRenderer
+import kotlinx.android.synthetic.main.activity_main.*
+import java.lang.Exception
+import java.util.Locale
+
+class MainActivity : AppCompatActivity() {
+
+  companion object {
+    private val TAG: String = MainActivity::class.java.simpleName
+  }
+
+  // create a service feature table and a feature layer from it
+  private val serviceFeatureTable: ServiceFeatureTable by lazy {
+    ServiceFeatureTable(getString(R.string.us_daytime_population_url))
+  }
+
+  // create the feature layer using the service feature table
+  private val featureLayer: FeatureLayer by lazy {
+    FeatureLayer(serviceFeatureTable)
+  }
+
+  override fun onCreate(savedInstanceState: Bundle?) {
+    super.onCreate(savedInstanceState)
+    setContentView(R.layout.activity_main)
+
+    val lineSymbol = SimpleLineSymbol(SimpleLineSymbol.Style.SOLID, Color.BLACK, 1.0f)
+    val fillSymbol = SimpleFillSymbol(SimpleFillSymbol.Style.SOLID, Color.YELLOW, lineSymbol)
+
+    featureLayer.apply {
+      // override the renderer
+      renderer = SimpleRenderer(fillSymbol)
+      opacity = 0.8f
+      maxScale = 10000.0
+    }
+
+    // create a map with the topographic basemap and add it to the basemap
+    mapView.map = ArcGISMap(Basemap.createTopographic()).apply {
+      // add the layer to the map
+      operationalLayers.add(featureLayer)
+      // zoom to a view point of the USA
+      initialViewpoint =
+        Viewpoint(Point(-11000000.0, 5000000.0, SpatialReferences.getWebMercator()), 100000000.0)
+    }
+
+  }
+
+  /**
+   * Handle the search intent from the search widget
```

Comment:
super picky, but for these types of javadoc descriptions (function descriptions, not inline comments), we have convention to treat them as sentences: just missing a period at the end here!